### ISSUE TITLE
CLZERO operand

### ISSUE THREAD

**Issue by andreas-abel** [intelxed/xed#163]
I think the implicit operand of the CLZERO instruction should be suppressed.

**Comment by andreas-abel** [comment]
The GNU assembler and NASM don't accept "CLZERO RAX" ("number of operands mismatch" and "invalid combination of opcode and operands"). objdump disassembles it as "clzero".

**Comment by andreas-abel** [comment]
BTW, are you sure that OrAX() is the correct type for the operand? The manual isn't clear about this, but I would assume that it should be ArAX(), as the register is supposed to hold a memory address.

**Comment by markcharney** [comment]
I agree the AMD doc is ambiguous and your interpretation makes sense. The operand should be ArAX(). I  do not have hardware to check. Possibly someone reading this does and has some spare time to check.

Aside: according to Peter, the nasm maintainer, "NASM standard is to accept, but not require or show in disassembly, implicit operands."  I reported the omission of the operand to him. I think a small fix for NASM is probably coming for this one eventually.  I didn't talk to the binutils guys yet. 

But yeah, I'll change it to ArAX() and I don't really mind much making the operand suppressed.  Thx.
